jjb/anax/anax.yml

```diff
@@ -0,0 +1,22 @@
+- job:
+    name: jenkins-job
+    project-type: pipeline
+    defaults: global
+    node: 'ubuntu2204-builder-aws-1c-2g'
+    pipeline-scm:
+      script-path: "Jenkinsfile"
+      scm:
+        - git:
+            url: "https://github.com/Al-tekreeti/anax.git"
```

Comment:
We shouldn't be building anything in your personal account. If you need to test outside of the main anax repo, we can create a clone/fork, but it needs to be in an LF-controlled org (such as open-horizon or lf-edge).

Reply:
Okay, I don't think there is a problem of using https://github.com/open-horizon/anax.git.